addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/MongoDbInjectionScanRule.java

```diff
@@ -251,7 +176,7 @@ public void scan(HttpMessage msg, NameValuePair originalParam) {
     @Override
     public void scan(HttpMessage msg, String param, String value) {
         boolean isBingo = false;
-        List<Integer> rtt = new ArrayList<>();
+        List<Integer> rtt = new java.util.ArrayList<>();
```

Comment:
Please properly use imports, I'm not going to keep commenting on this. Please assume that you should import and not fully qualify declarations throughout.

Reply:
What's the rational? (Just curiosity.)

Reply:
Consistency with the rest of the code base. There are a very view places where we have class conflicts that require this.

Reply:
I was referring to the code change not your comment, sorry.